modules/hal_rpi_pico/pico/version.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2021, Yonatan Schachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _PICO_VERSION_H
+#define _PICO_VERSION_H
+
+#define PICO_SDK_VERSION_MAJOR 1
+#define PICO_SDK_VERSION_MINOR 3
+#define PICO_SDK_VERSION_REVISION 0
+#define PICO_SDK_VERSION_STRING "1.3.0"
```

Comment:
What's the plan for keeping this up to date with HAL changes in west.yml? Why isn't this coming from the HAL module repository instead?

Reply:
This isn't really required, I'll remove it